zulipterminal/ui_tools/buttons.py

```diff
@@ -18,11 +18,12 @@ class TopButton(urwid.Button):
     def __init__(self, controller: Any, caption: str,
                  show_function: Callable[..., Any], width: int,
                  prefix_character: Union[str, Tuple[Any, str]]='\N{BULLET}',
-                 text_color: Optional[str]=None,
+                 text_color: Optional[str]=None, is_topic: bool=False,
```

Comment:
@neiljp This maybe slightly tricky. Generalizing this to the other buttons I feel is not so easy and after doing a quick mock-up the other don't benefit much from space increse. I pushed a newer version which uses a simpler logic rather than passing a flag from child -> parent, which may solve your worry regarding that.

I had tried before to use `prefix_character` for account spacing. The problem with that is the on hover/focus of the button, there are to large gaps -> after vs current
![after](https://user-images.githubusercontent.com/30312566/60757674-31242100-a02b-11e9-90e4-459ab567f3b4.png)
![current](https://user-images.githubusercontent.com/30312566/60757676-32ede480-a02b-11e9-9987-6fa4d240bded.png)